kondo-lib/src/lib.rs

```diff
@@ -237,11 +242,13 @@ impl Project {
             ProjectType::Elixir => PROJECT_ELIXIR_NAME,
             ProjectType::Swift => PROJECT_SWIFT_NAME,
             ProjectType::Gradle => PROJECT_GRADLE_NAME,
+            ProjectType::Zig => PROJECT_ZIG_NAME,
         }
     }
 
     /// Deletes the project's artifact directories and their contents
     pub fn clean(&self) {
+        println!("{:?}", self.artifact_dirs());
```

Comment:
Looks like this was left in by accident :)

Reply:
Oops, sorry. I'm at an airport with 2hrs of sleep. Will fix ASAP!

Reply:
No rush :)

Reply:
PR is updated!